shared/actions/provision.js

```diff
@@ -1,5 +1,6 @@
 // @flow
 import * as Constants from '../constants/provision'
+import {_getNavigator} from '../constants/router2'
```

Comment:
same comment as in provisioning. lets not bleed this into the sagas. if the sagas need to know something they should just dispatch actions. i think exposing them directly to the raw nav isn't a good idea